### PR TITLE
[doc] Update links in the readme file.

### PR DESCRIPTION
**Pull request by rexwangcc**
Related issue = #

Questions: 
- I'm not sure if we want to hide the link to `Hub` for now? 
- Is `contact` the right email address for security disclosures?
- In order to fix https://github.com/taichi-dev/docs.taichi.graphics/pull/71, I loosened the ignore file and added the missing JSON metadata and image back to docs folder. Was there a reason to ignore JPG and JSON files repo-wide at the first place?

### CODE REVIEW

**Review by ailzhang — APPROVED** [review]
Thanks for updating the links!

**Comment by rexwangcc** [comment]
Looks like the PR is stuck due to the fact I skipped CI checks which are "required statuses before merging" 🤦

**Review by ailzhang — APPROVED** [review]
I'm not an expert in frontend deployment but this looks good :P 
Stamping to unblock the cron job :D

File: README.md
Comment (FantasyVR) [inline README.md:3]: The users could interact with some interesting Taichi programs on their web browser without installing Taichi. I think we don't need to show this right now. The `Taichi Zoo` could replace the `playground` in the future.

File: .gitignore
Comment (FantasyVR) [inline .gitignore:81]: Thank you very much @rexwangcc. I'm sorry I didn't double-check whether the `.json` files are copied into the main repo.

**Review by FantasyVR — APPROVED** [review]

File: README.md
Comment (rexwangcc) [reply README.md:3]: Thanks! Let's comment it out for now then.

File: .gitignore
Comment (rexwangcc) [reply .gitignore:81]: No worries at all, I didn't realize it either until the CI cries, CI for the win!

**Pull request merged**
